### PR TITLE
Unindent over-indented functions

### PR DESCRIPTION
**Pull request by adamrp**
For some reason (probably erroneous copy and paste) many functions in
qiita_db/util.py were indented too much. No changes were made here other than
to correct the indentation level of these functions.

### CODE REVIEW

File: qiita_db/util.py [hunk 1]
```diff
@@ -255,53 +255,53 @@ def get_emp_status(key='emp_status'):
...
-        chars = ''.join((ascii_letters, digits))
-        if punct:
-            chars = ''.join((chars, punctuation))
-        return ''.join(choice(chars) for i in range(length))
+    Parameters
+    ----------
+    length: int
+        Length of string to return
+    punct: bool, optional
+        Include punctiation as well as letters and numbers. Default True.
```
Comment (wasade) [inline qiita_db/util.py]: punctuation

File: qiita_db/util.py [hunk 2]
```diff
@@ -255,53 +255,53 @@ def get_emp_status(key='emp_status'):
...
-        if hashedpw is None:
-            hashedpw = gensalt()
-        else:
-            hashedpw = hashedpw.encode('utf-8')
-        password = password.encode('utf-8')
-        output = hashpw(password, hashedpw)
-        if isinstance(output, bytes):
-            output = output.decode("utf-8")
-        return output
+    """ Hashes password
```
Comment (wasade) [inline qiita_db/util.py]: Extra space

File: qiita_db/util.py [hunk 3]
```diff
@@ -255,53 +255,53 @@ def get_emp_status(key='emp_status'):
...
+    -----
+    Relies on bcrypt library to hash passwords, which stores the salt as
+    part of the hashed password. Don't need to actually store the salt
+    because of this.
+    """
+    # all the encode/decode as a python 3 workaround for bcrypt
+    if hashedpw is None:
+        hashedpw = gensalt()
+    else:
+        hashedpw = hashedpw.encode('utf-8')
```
Comment (wasade) [inline qiita_db/util.py]: why is this necessary? the encrypting here is much more complex than what we're doing on american gut

File: qiita_db/util.py [hunk 4]
```diff
@@ -869,66 +869,66 @@ def filepath_ids_to_rel_paths(filepath_ids):
...
-
-        Parameters
-        ----------
-        value : str
-            The string value to convert
-        table : str
-            The table that has the conversion
-        conn_handler : SQLConnectionHandler, optional
-            The sql connection object
+    """Converts a string value to it's corresponding table identifier
```
Comment (wasade) [inline qiita_db/util.py]: its

File: qiita_db/util.py [hunk 5]
```diff
@@ -869,66 +869,66 @@ def filepath_ids_to_rel_paths(filepath_ids):
...
-
-        Parameters
-        ----------
-        value : int
-            The id value to convert
-        table : str
-            The table that has the conversion
-        conn_handler : SQLConnectionHandler, optional
-            The sql connection object
+    """Converts an id value to it's corresponding string value
```
Comment (wasade) [inline qiita_db/util.py]: its

**Comment by wasade** [comment]
few comments

File: qiita_db/util.py (hunk 3 above)
Comment (squirrelo) [inline qiita_db/util.py]: We are using the bcrypt library directly, which only works with utf-8 data. Americangut is using a wrapper around bcrypt library.

**Comment by squirrelo** [comment]
Reviewed and no comments added on top of @wasade 's

File: qiita_db/util.py (hunk 3 above)
Comment (wasade) [inline qiita_db/util.py]: question was in regards to dealing with the salt

Comment (squirrelo) [inline qiita_db/util.py]: Ah, this is also because we're using the bcrypt library directly. the `hashpw` needs the string and a salt. This salt can either be a new salt if it's not comparing anything, or the hashed password and it pulls the hash from that password for comparison later.

Comment (adamrp) [inline qiita_db/util.py]: @wasade or @squirrelo I'm going to leave this as is right now, but does one of you want to create an issue about this?

Comment (squirrelo) [inline qiita_db/util.py]: Good as is, just no wrapper from other package. No need for issue unless we want to change to the same package as American Gut.
